src/components/Layout.js

```diff
@@ -85,4 +85,14 @@ exports.addSystemTestSupport = function(){
         browser.get(url);
         browser.sleep(3000);
     });
+};
+
+exports.acceptAlert = function(){
+    browser.sleep(1000);
+    if(browser.browserName === 'chrome'){
+        browser.switchTo().alert().accept();
+    }
+    if(browser.browserName === 'phantomjs'){
```

Comment:
umm if the browser is not chrome it will continue to the next 'else if' and if it won't be phantomjs it wont run, I just think it should work as switch-case  but it's not critical cause we don't mind the performance